### ISSUE TITLE
Launch Tk Script Not Working

### ISSUE THREAD

**Issue by msauvayre** [DonJayamanne/pythonVSCode#177]
Hi,

I have an issue with a script who use Tk. I use an icon with a file named INL.ico, but I have this error :
File "c:\Python27\lib\lib-tk\Tkinter.py", line 1691, in wm_iconbitmap
    return self.tk.call('wm', 'iconbitmap', self._w, '-default', default)
_tkinter.TclError: bitmap "Resources\INL.ico" not defined.

Except that when I launching the script in debug mode, it working, and when I launch thru the terminal, it working to.

Since two days, I searching the solution, I think it's probably a parameter to fix in tasks.json or launch.json but I don't found the solution.

Do you have some idea about this issue ?

Regards,
Max

**Comment by DonJayamanne** [comment]
Please could you upload the rest of the code.

**Comment by msauvayre** [comment]
Hello, 

here it is :
[GUI.txt](https://github.com/DonJayamanne/pythonVSCode/files/324970/GUI.txt)

For information, i tried many solutions : use relpath instead of the name only, slash and double slash, back slash ans double back slash...
Nothing works...

Thanks for your help
Max

**Comment by msauvayre** [comment]
Re,

I locate the problem. It's on tasks.json, especially "args" parameter.
When I make getcwd in a subfolder of the workspace, it give the workspace instead of subfolder.
But I can't find the correct parameter to put in.

So, in my script, I completed the icon path and it worked.

You have a clue to fix that ?

**Comment by DonJayamanne** [comment]
@msauvayre, apologies for not getting back to you on this. Are you still facing the same issue?

**Comment by msauvayre** [comment]
Hi Don,

my apologies to, in this summer time, I have more work than usual because of vacation days, and less time for the others things.

This aside, I managed this problem. Originally, I worked with Eclipse who consider workspace as a main folder, and projects are in subfolder of him.

But in VS code, project should be considered an individual workspace as VS Code conceives. So, with opening the project folder directly in VS Code and not the main workspace who contain it.

From there, everything worked. So I don't think it's an issue, it's more a bad use of VS Code and your excellent plugin that I did.

Thanks for your help.
Max
